Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -514,8 +514,7 @@ void BluetoothReal::LoadLinkKeys()
     if (index == std::string::npos)
       continue;
 
-    bdaddr_t address;
-    Common::StringToMacAddress(pair.substr(0, index), address.data());
+    bdaddr_t address = Common::StringToMacAddress(pair.substr(0, index)).value();
```

Comment:
@leoetlino just one thing I noticed while making changes, but is it safe to assume these addresses will always be valid like the previous code was doing? It doesn't seem like we sanitize the link keys anywhere beforehand to ensure they're valid MAC addresses before using them.

Reply:
Hmm, good catch -- we should probably ignore the link key entry if the MAC address is invalid. (Though that can only normally happen if the user corrupted their own config file on purpose)

Reply:
Thanks, I added a commit to skip it if that situation ever happens; may as well protect users from themselves.